tests/model/test_core.py

```diff
@@ -1786,15 +1782,24 @@ def test_copy_model(self, copy_method) -> None:
                 samples=1, random_seed=42
             )
 
-        simple_model_prior_predictive_mean = simple_model_prior_predictive["prior"]["y"].mean(
-            ("chain", "draw")
-        )
-        copy_simple_model_prior_predictive_mean = copy_simple_model_prior_predictive["prior"][
+        simple_model_prior_predictive_val = simple_model_prior_predictive["prior"]["y"].values
+        copy_simple_model_prior_predictive_val = copy_simple_model_prior_predictive["prior"][
             "y"
-        ].mean(("chain", "draw"))
+        ].values
 
-        assert np.isclose(
-            simple_model_prior_predictive_mean, copy_simple_model_prior_predictive_mean
+        assert simple_model_prior_predictive_val == copy_simple_model_prior_predictive_val
+
+        with copy_simple_model:
+            z = pm.Deterministic("z", copy_simple_model["alpha"] + 1)
+            copy_simple_model_prior_predictive = pm.sample_prior_predictive(
+                samples=1, random_seed=42
+            )
```

Comment:
You can do this above, and call `sample_prior_predictive` only once for the `copy_simple_model`